data/json/items/ammo.json

```diff
@@ -1,4 +1,22 @@
 [
+  {
+    "type": "AMMO",
+    "id": "air",
+    "category": "spare_parts",
+    "price": 12000,
+    "name": "air",
+    "name_plural": "air",
+    "symbol": "=",
+    "color": "yellow",
+    "description": "Some free-floating compressed air.  This can be reloaded into rechargable air cylinders, but can never be unloaded.",
```

Comment:
we have the `gas` phase in code, we should be using it for something like this.

Reply:
You want me to just put in
`"phase": "gas",`
?

Reply:
> we have the `gas` phase in code, we should be using it for something like this.

Could you explain what you mean? I haven't seen that particular use of it in the JSON so I'm not sure what it's for. Is it there to prevent the creation of an "air" item upon destruction of the cylinder/container like for example with the destruction of a car battery/battery charges?